pkgs/build-support/setup-hooks/strip.sh

```diff
@@ -68,6 +68,11 @@ stripDirs() {
         striperr="$(mktemp 'striperr.XXXXXX')"
         # Do not strip lib/debug. This is a directory used by setup-hooks/separate-debug-info.sh.
         find $paths -type f -a '!' -path "$prefix/lib/debug/*" -print0 |
+            # Make sure we process files under symlinks only once. Otherwise
+            # 'strip` can corrupt files when writes to them in parallel:
+            #   https://github.com/NixOS/nixpkgs/issues/246147#issuecomment-1657072039
+            xargs -r -0 -n1 -- realpath -z | uniq -z |
```

Comment:
`uniq` only removes *adjacent* duplicates. Try `sort -u`.

Reply:
Oh, great catch! I mainly tested on the libraries, which usually go woth `.so / so.1.2.3` in succession.

Changed to `sort -u -z`.

Thank you!